Util/Xamarin.Build.Download/source/Xamarin.Build.Download/BaseXamarinBuildResourceRestore.cs

```diff
@@ -104,9 +104,11 @@ public override bool Execute ()
 		bool MergeResources (IAssemblyResolver resolver, string originalAsmPath, string mergedAsmPath, string assemblyName, List<ITaskItem> resourceItems)
 		{
 			var disposeList = new List<IDisposable> ();
```

Comment:
Nice suggestion!  I would like to use that however I just found that it requires that I ship additional dlls in the NuGet just for a simple concept.